code/style.css

```diff
@@ -1,7 +1,369 @@
+  grid-template-rows: 100px 100px 100px;
+  justify-items: center;
+}
+  
+h1 {
+  color: white;
+  font-size: 80px;
+  writing-mode: vertical-lr;
+  color: white;
+  margin: 0;
+  text-align: center;
+  line-height: .9;
+}
+
+.rotate {
+  transform: rotate(180deg);
+  color: white;
+}
+
+.small-text {
+  font-size: 20px;
+  color:black;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  background-color: white;
+}
+
+.container {
+  background-color: white;
+  text-align: center;
+}
+
+.form {
+display: flex;
+flex-direction: column;
+margin-bottom: 30px;
+height: auto;
+margin: 0 auto;
+-webkit-box-shadow: 17px 18px 10px 2px rgba(0,0,0,0.2);
+-moz-box-shadow: 17px 18px 10px 2px rgba(0,0,0,0.2);
+box-shadow: 17px 18px 10px 2px rgba(0,0,0,0.2);
+width: 600px;
+border: 2px solid black;
+}
+
+.form h3 {
+  text-align: center;
+  color: black;
+}
+
+label {
+  color: black;
+  margin: 15px;
+  padding: 10px;
+  width: 100px;
+  font-size: 15px;
+}
+
+.form-group {
+display: flex;
+align-items: center;
+justify-content: center;
+}
+
+.input-text {
+width: 300px;
+height: 30px;
+border-radius: 10px;
+cursor: pointer;
+}
+
+.submit-button {
+background-color: black;
+color: white;
+cursor: pointer;
+padding: 16px 32px;
+margin: 4px 2px;
+border: none;
+}
+
+
+.accordion {
+background-color: white;
+padding: 18px;
+width: 100%;
+text-align: center;
+border: none;
+outline: none;
+transition: 0.4s;
+font-size: 20px;
+}
+
+.accordion h1 {
+writing-mode: unset;
+color: black;
+background-color: white;
+text-align: center;
+font-size: 30px;
+margin-bottom: 10px;
+}
+
+.accordion-header {
+cursor: pointer;
+background-color: black;
+color: white;
+}
+
+.accordion-element {
+margin-bottom: 15px;
+}
+
+.accordion-text {
+color: black;
+}
+
+.accordion-text-container.active {
+height: 150px;
+}
+
+.accordion-text-container {
+height: 0;
+overflow: hidden;
+transition: height 0.3s ease;
+}
+
+.accordion-text {
+padding: 18px;
+background-color: white;
+}
+
+footer {
+text-align: center;
+position: fixed;
+width: 100%;
+height: 30px;
+background-color: black;
+color: white;
+left: 0;
+bottom: 0;
+font-size: 20px;
+}
+
+@import url('https://fonts.googleapis.com/css?family=Playfair+Display:400,400i,700,700i,900,900i|Poppins:300,400,500,600,700,800,900');
+@import url('https://fonts.googleapis.com/css?family=Forum');
+
+@import url('https://fonts.googleapis.com/css?family=Cinzel:400,700,900|Josefin+Slab:100,100i,300,300i,400,400i,600,600i,700,700i|Marcellus|Nanum+Myeongjo:400,700,800|Old+Standard+TT:400,400i,700|Prata|Vidaloka');
+
+
+
+@import url("https://fonts.googleapis.com/css?family=Playfair+Display:400,400i,700,700i,900,900i|Poppins:300,400,500,600,700,800,900");
+@import url("https://fonts.googleapis.com/css?family=Forum");
+@import url("https://fonts.googleapis.com/css?family=Cinzel:400,700,900|Josefin+Slab:100,100i,300,300i,400,400i,600,600i,700,700i|Marcellus|Nanum+Myeongjo:400,700,800|Old+Standard+TT:400,400i,700|Prata|Vidaloka");
```

Comment:
The @import should be at the top of your CSS file and these look like duplicates?